proxy/proxy.go

```diff
@@ -1205,7 +1205,7 @@ func (p *Proxy) serveResponse(ctx *context) {
 	p.tracing.logStreamEvent(ctx.proxySpan, StreamBodyEvent, strconv.FormatInt(n, 10))
 	if err != nil {
 		p.metrics.IncErrorsStreaming(ctx.route.Id)
-		p.log.Errorf("error while copying the response stream: %v", err)
+		p.log.Debugf("error while copying the response stream: %v", err)
```

Comment:
are you sure that we don't get this error in cases which we need?

Reply:
My point is that this message is useless for debugging in production because it lacks details of the request.
For monitoring purposes `streaming_errors` metric and/or opentracing could be used.

Reply:
> Then we should drop the log

This still could be useful for local single request debugging. Also there is a test that depends on this log line and I did not want to spend much time refactoring it

Reply:
:+1: